docs/element-definitions.md

```diff
@@ -1285,11 +1289,12 @@ transitions.
 #### Classifications
 
 - `"api"` - Category is a API top-level group.
-- `"resourceGroup"` - Category is a set of resources.
+- `"authSchemes"` - Category is a group of authentication and authorization scheme definitions.
 - `"dataStructures"` - Category is a set of data structures.
+- `"hosts"` - Category is a set of servers.
```

Comment:
I still wonder whether we need this category at all. Perhaps the `host` resources can just live in the toplevel category element? @kylef , what do you think?

Reply:
We should explain a bit more about what the hosts classification is for the category. Although I'd delegate that to a reference to the resource classifications.

Perhaps something along the lines of:

```suggestion
- `"hosts"` - Category of [Resource][] which implicitly contain the Resource `host` classification. See the classifications in [Resource][] for further semantics.
```